apps/condo/domains/ticket/schema/ExportTicketsService.js

```diff
@@ -63,7 +64,7 @@ const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
                         operator: ticket.operator || ticket.createdBy || '',
                         executor: ticket.executor || '',
                         assignee: ticket.assignee || '',
-                        comments: ticket.TicketComment.join('\n'),
+                        comments: comments.map(comment => comment.split(':').pop()).join('\n' + '—'.repeat(20) + '\n'),
```

Comment:
Please, explain in comment, what this 20 dashes are used for.

Reply:
It's a separator for comments - will make comment

Reply:
Fixed